configurations/za-prod.ts

```diff
@@ -0,0 +1,14 @@
+import { Configuration } from './types';
+import { config as zaStaging } from './za-staging';
+
+const accountSid = 'AC988e78b713be4a04246b39835de37ad4';
+const flexFlowSid = 'FOacabd43f752e8e0ca3d2466e90637095';
```

Comment:
I think we can move these values to a secret.js file and populate them from GitHub actions. But of course, we can handle that later in another Jira ticket.